src/Core/System/Language/LanguageDefinition.php

```diff
@@ -97,20 +98,29 @@ public function getEntityClass(): string
         return LanguageEntity::class;
     }
 
+    /**
+     * @return array<string, mixed>
+     */
+    public function getDefaults(): array
+    {
+        return ['active' => 1];
```

Comment:
shouldn't this be 

```suggestion
        return ['active' => true];
```

?

Reply:
I was totally not sure about this, since I saw both. Gladly will adjust this